### ISSUE TITLE
Als medewerker van de gemeente Den Haag wil ik een overzicht van mijn zaken voor een aantal verschillende toegestane zaaktypes en deze informatie in 1x opvragen via de Zaken API.

### ISSUE THREAD

**Issue by VerbeekIT** [VNG-Realisatie/gemma-zaken#2076]
Als medewerker van de gemeente Den Haag wil ik een overzicht van mijn zaken voor een aantal verschillende toegestane zaaktypes en deze informatie in 1x opvragen via de Zaken API.

Wat ik als gemeente wil is dat ik de toegestane zaaktypes per medewerker en per rol kan bepalen, en de lijst van toegestane zaaktypes automatisch kan toevoegen aan zaaklijst opvragingen.

Wat ik als gemeente niet wil is dat de medewerker zicht heeft op zaken van zaaktypes die voor die medewerker niet zichtbaar mogen zijn, en wat ik als gemeente ook niet wil is dat de zaaklijst opvragingen per zaaktype uitgevoerd moeten worden omdat dan de medewerker te lang moet wachten en er dan meerdere zaaklijst API calls moeten worden samengevoegd.

**Comment by alextreme** [comment]
Zelfde issue als #2072

**Comment by michielverhoef** [comment]
Niet helemaal denk ik. 

De achterliggende vraag is: Gaat het om verschillende versies van 1 zaaktype of om echt verschillende zaaktypen (bv parkeervergunning, kapvergunning, buurtfeest)? 
Zie ook https://github.com/VNG-Realisatie/gemma-zaken/issues/2072#issuecomment-1260450727 
@VerbeekIT

**Comment by VerbeekIT** [comment]
Ja. Het gaat hier om zaken van een verschillend zaaktype. Niet om versie verschillen.

**Comment by michielverhoef** [comment]
@VerbeekIT Waarmee is dit issue dan opgelost volgens jou?

**Comment by michielverhoef** [comment]
> Ja. Het gaat hier om zaken van een verschillend zaaktype. Niet om versie verschillen.

In dat geval denk ik dat kunnen filteren op meerdere url's geen handige oplossing is.  Het lijkt me dat een medewerker dan toegang moet kunnen hebben tot verschillende versies van verschillende zaaktypen.

Dus Zaaktype A verzie 1, Zaaktype A versie 2, Zaaktype B versie 1, Zaaktype C versie 1, Zaaktype C versie 2 etc.

Dan zou ik toch de neiging krijgen om te willen filteren op zaaktypeidentificatie ipv de url.
